.gitignore

```diff
@@ -17,7 +17,7 @@ todo.txt
 /alpine-keys.apk
 /alpine
 
-/target
```

Comment:
Let's revert this one. It's not needed with workspaces and I believe it may be a problem because of ignoring such a generic name.